website/www/site/content/en/documentation/io/built-in/snowflake.md

```diff
@@ -0,0 +1,738 @@
+  "--externalLocation=<GCS BUCKET URL STARTING WITH GS://>",
+      Example: --tempLocation=gs://my_bucket/temp/
+]' --no-build-cache
+{{< /highlight >}}
+
+Where all parameters are starting with “--”, they are surrounded with double quotation and separated with comma:
+
+- `--serverName=<SNOWFLAKE SERVER NAME>`
+  - Specifies the full name of your account (provided by Snowflake). Note that your full account name might include additional segments that identify the region and cloud platform where your account is hosted. 
+  - Example: `--serverName=xy12345.eu-west-1.gcp..snowflakecomputing.com`
+
+- `--username=<SNOWFLAKE USERNAME>`
+  - Specifies the login name of the user.
+  - Example: `--username=my_username`
+
+- `--password=<SNOWFLAKE PASSWORD>`
+  - Specifies the password for the specified user.
+  - Example: `--password=my_secret`
+
+- `--schema=<SNOWFLAKE SCHEMA>`
+  - Specifies the schema to use for the specified database once connected. The specified schema should be an existing schema for which the specified user’s role has privileges.
+  - Example: `--schema=PUBLIC`
+
+- `--table=<SNOWFLAKE TABLE IN DATABASE>`
+  - Example: `--table=MY_TABLE`
+
+- `--database=<SNOWFLAKE DATABASE>`
+  - Specifies the database to use once connected. The specified database should be an existing database for which the specified user’s role has privileges.
+  - Example: `--database=MY_DATABASE`
+
+- `--storageIntegrationName=<SNOWFLAKE STORAGE INTEGRATION NAME>`
+  - Name of storage integration created in [Snowflake](https://docs.snowflake.com/en/sql-reference/sql/create-storage-integration.html) for a cloud storage of choice.
+  - Example: `--storageIntegrationName=my_google_integration`
+
+## Running pipelines on Dataflow
+By default, pipelines are run on [Direct Runner](https://beam.apache.org/documentation/runners/direct/) on your local machine. To run a pipeline on [Google Dataflow](https://cloud.google.com/dataflow/), you must provide the following Pipeline options:
+
+- `--runner=DataflowRunner`
+  - The Dataflow’s specific runner.
+
+- `--project=<GCS PROJECT>`
+  - Name of the Google Cloud Platform project.
+
+- `--stagingBucketName=<GCS BUCKET NAME>`
+  - Google Cloud Services bucket where the Beam files will be staged.
+
+- `--maxNumWorkers=5`
+  - (optional) Maximum number of workers.
+
+- `--appName=<JOB NAME>`
+  - (optional) Prefix for the job name in the Dataflow Dashboard.
+
+More pipeline options for Dataflow can be found [here](https://beam.apache.org/releases/javadoc/2.17.0/org/apache/beam/runners/dataflow/options/DataflowPipelineOptions.html).
+
+**Note**: To properly authenticate with Google Cloud, please use [gcloud](https://cloud.google.com/sdk/gcloud/) or follow the [Google Cloud documentation](https://cloud.google.com/docs/authentication/).
+
+**Important**: Please acknowledge [Google Dataflow pricing](Important: Please acknowledge Google Dataflow pricing).
+
+###Running pipeline templates on Dataflow
```

Comment:
This is also not yet in this release @purbanow

Reply:
True , we must remvoe it

Reply:
Removed